tcf_website/models/models.py

```diff
@@ -369,6 +369,8 @@ class Course(models.Model):
     # Course number. Required.
     number = models.IntegerField(
         validators=[MinValueValidator(0), MaxValueValidator(99999)])
+    # Prerequisite. Optional.
+    pre_req = models.TextField(blank=True)
```

Comment:
Added this so it is easier for me to refer to it in `course_description_without_prerequisite`

Reply:
Also shaky about my use of `pre_req` and `prerequisite` I should probably standardize that right?